app/charts/shared/legend-color.tsx

```diff
@@ -67,44 +67,58 @@ type ItemStyleProps = {
   usage: LegendItemUsage;
 };
 
-const useItemStyles = makeStyles<Theme, ItemStyleProps>((theme) => ({
-  legendItem: {
-    position: "relative",
-    justifyContent: "flex-start",
-    alignItems: "flex-start",
-    fontSize: ({ usage }) =>
-      ["legend", "colorPicker"].includes(usage)
-        ? theme.typography.body2.fontSize
-        : theme.typography.caption.fontSize,
-    fontWeight: theme.typography.fontWeightRegular,
-    color: theme.palette.grey[700],
+const useItemStyles = makeStyles<Theme, ItemStyleProps>((theme) => {
+  const getSizeAdjustConstant = (
+    symbol: LegendSymbol,
+    usage: LegendItemUsage
+  ) => {
+    if (usage === "colorPicker" || symbol === "circle") {
+      return 1.5;
+    }
+
+    return 1;
+  };
```

Comment:
Nit: I guess this could be implemented via a CSS variable ? This way you do not generate other styles.


```
legendItem: {
     width: ({ symbol, usage }) =>
        symbol === "circle" ? "50%" : 0,
          `calc(0.5rem * var(--size-adjust, 1))`,
},
colorPicker: {
  '--size-adjust': '1.5'
},

circle: {
  '--size-adjust': '1.5'
},
```

Reply:
I've never thought about this, thanks for this suggestion 💯 Done in https://github.com/visualize-admin/visualization-tool/pull/1435/commits/12b19801d8308fd1412d391fc5d1f63d1435bd19 :)